python/ray/air/result.py

```diff
@@ -57,10 +57,16 @@ def __repr__(self):
         from ray.tune.result import AUTO_RESULT_KEYS
 
         shown_attributes = {k: self.__dict__[k] for k in self._items_to_repr}
+        if self.error:
+            shown_attributes["error"] = type(self.error).__name__
+        else:
+            shown_attributes.pop("error")
 
         if self.metrics:
             shown_attributes["metrics"] = {
                 k: v for k, v in self.metrics.items() if k not in AUTO_RESULT_KEYS
             }
+
         kws = [f"{key}={value!r}" for key, value in shown_attributes.items()]
-        return "{}({})".format(type(self).__name__, ", ".join(kws))
+        kws_repr = "  " + ",\n  ".join(kws)
+        return "{}(\n{}\n)".format(type(self).__name__, kws_repr)
```

Comment:
It's very hard to imagine how this looks like, can we add a comment what we expect here?